lib/helpers/syncGSheet.js

```diff
@@ -551,7 +571,27 @@ module.exports = class ChanakyaGSheetSync {
             if ( dbKey.totalMarks ) {
                 row['testScore'] = dbKey.totalMarks
             }
-
+            
+            // auto change rqcStatus and Actual Stage.
```

Comment:
What's the significance this statement?  Why is Actual Stage being changed to `Pending Callback for Query` when the stage is `testPassed`. Also it will be helpful if you can add comments to the code you write so it's easy to understand the logic you used.